src/vcpkg.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "antares-simulator",
+  "version-string": "9.1.0",
```

Comment:
i don't if it's possible, but i will be nice to update the version automatically